pyproject.toml

```diff
@@ -19,6 +19,8 @@ dynamic = ["version"]
 requires-python = ">=3.7"
 dependencies = [
     "python-on-whales>=0.51.1",
+    "requests>=2.28.1",
+    "boto3>=1.24.73",
 ]
```

Comment:
could you move the boto dependency to an `optional-dependencies` block as for the other containers?